tagpack/tagstore.py

```diff
@@ -227,7 +332,7 @@ def get_quality_measures(self, currency="") -> float:
         keys = ["count", "avg", "stddev"]
         return {keys[i]: v for row in self.cursor.fetchall() for i, v in enumerate(row)}
 
-    def calculate_quality_measures(self) -> float:
+    def calculate_quality_measures(self) -> dict:
```

Comment:
Is this quality view something we want to keep up to date in general? If so maybe add call to this to tp-tool tagstore refresh_views